### PR TITLE
Added additional unit tests to test_integrals()

### PR DESCRIPTION
**Pull request by jremes-foss**
Hello.

The test_integrals was lacking some elementary functions unit-test - wise, so I decided to add few more, just to cover the trigonometry.

Thank you.

### CODE REVIEW

File: sympy/integrals/tests/test_integrals.py [hunk 1]
```diff
@@ -134,6 +134,10 @@ def test_integration():
     assert integrate(t**2 + 5*t - 8, (t, 0, x)) == x**3/3 + 5*x**2/2 - 8*x
     assert integrate(x**2, x) == x**3/3
     assert integrate((3*t*x)**5, x) == (3*t)**5 * x**6 / 6
+    assert integrate(sin(x), x) == -cos(x)
```
Comment (debugger22) [inline sympy/integrals/tests/test_integrals.py]: `sympy/integrals/tests/test_trigonometry.py:13:    assert trigintegrate(sin(x), x) == -cos(x)`

File: sympy/integrals/tests/test_integrals.py [hunk 2]
```diff
@@ -134,6 +134,10 @@ def test_integration():
...
+    assert integrate(cos(x), x) == sin(x)
```
Comment (debugger22) [inline sympy/integrals/tests/test_integrals.py]: `sympy/integrals/tests/test_trigonometry.py:14:    assert trigintegrate(cos(x), x) == sin(x)`

**Comment by Newman101** [comment]
Ok, it seems that the unit tests were there after all.

**Comment by toolforger** [comment]
@Newman101 is there anything left to be done about this? If not, please close it.

**Comment by Newman101** [comment]
Nope, I'm closing this one.

**Pull request closed without merging**